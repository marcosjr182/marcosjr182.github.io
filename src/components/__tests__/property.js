import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

import Property from '../property'

const item = { name: 'Water', qty: 5 }
let wrapper = shallow(<Property name={item.name} value={item.qty} />)

describe('Property component', () => {

  it('should render a property correctly', () => {
    expect(toJson(wrapper)).toMatchSnapshot()
  })

  it('should accept an optional columns props', () =>{
    wrapper = shallow(<Property name={item.name} value={item.qty} columns='3' /> )
    expect(toJson(wrapper)).toMatchSnapshot()
  })

})
