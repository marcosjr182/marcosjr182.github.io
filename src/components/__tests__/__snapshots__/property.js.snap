exports[`Property component should accept an optional columns props 1`] = `
<div
  className="col-xs-3 item Water">
  <div
    className="col-xs-12 name">
     
    Water
     
  </div>
  <div
    className="col-xs-12 qty">
     
    5
     
  </div>
</div>
`;

exports[`Property component should render a property correctly 1`] = `
<div
  className="col-xs-6 item Water">
  <div
    className="col-xs-12 name">
     
    Water
     
  </div>
  <div
    className="col-xs-12 qty">
     
    5
     
  </div>
</div>
`;
